vllm/worker/habana_worker.py

```diff
@@ -162,7 +162,7 @@ def initialize_cache(self, num_gpu_blocks: int,
         self.cache_config.num_cpu_blocks = num_cpu_blocks
 
         self._init_cache_engine()
-        self._warm_up_model()
+        # self._warm_up_model()
```

Comment:
No need to comment this; we can skip the warmup by setting the below environment variable 

VLLM_SKIP_WARMUP=true